deploy/helm/charts/README.md

```diff
@@ -11,10 +11,10 @@ A Helm chart for openebs dynamic nfs provisioner. This chart bootstraps OpenEBS
 
 | Name | Email | Url |
 | ---- | ------ | --- |
-| akhilerm | akhil.mohan@mayadata.io |  |
 | kiranmova | kiran.mova@mayadata.io |  |
-| prateekpandey14 | prateek.pandey@mayadata.io |  |
+| mayank-patel | mayank.patel@mayadata.io |  |
 | rahulkrishnanra | rahulkrishnanfs@gmail.com |  |
+| sai-chaithanya | sai.chaithanya@mayadata.io |  |
```

Comment:
Looks like it is expecting for GitHub ID
```suggestion
|  mittachaitu  | sai.chaithanya@mayadata.io |  |
```

Reply:
yes